forge-game/src/main/java/forge/game/ability/effects/VillainousChoiceEffect.java

```diff
@@ -0,0 +1,44 @@
+package forge.game.ability.effects;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+import forge.game.ability.AbilityUtils;
+import forge.game.ability.SpellAbilityEffect;
+import forge.game.player.Player;
+import forge.game.spellability.SpellAbility;
+
+import java.util.List;
+
+
+public class VillainousChoiceEffect extends SpellAbilityEffect {
+    @Override
+    public void resolve(SpellAbility sa) {
+        final List<SpellAbility> abilities = Lists.newArrayList(sa.getAdditionalAbilityList("Choices"));
+        final int amount = extractAmount(sa);
+        String prompt = sa.getParamOrDefault("ChoicePrompt", "Villainous Choice by " + sa.getActivatingPlayer());
+
+        for (Player p : getDefinedPlayersOrTargeted(sa)) {
+            int choiceAmount = p.getAdditionalVillainousChoices() + 1;
```

Comment:
I think with the Valeyard wording it's not linear like the voting 🤔 
but instead if you control this twice it would go from 1->2->4
so probably `2^p.getAdditionalVillainousChoices()`?

Reply:
I would prefer to use Replacement Effects for this, but I will check that out later

Reply:
If thats what the math is supposed to be I'm happy to switch it, I didn't see that in the faq anywhere?

Reply:
yea I asked judges and have been proven wrong 🍌